diff -up sqlite-autoconf-3430000/sqlite3.c.omv~ sqlite-autoconf-3430000/sqlite3.c
--- sqlite-autoconf-3430000/sqlite3.c.omv~	2023-08-25 16:18:40.382219932 +0200
+++ sqlite-autoconf-3430000/sqlite3.c	2023-08-25 16:19:04.378455941 +0200
@@ -176813,7 +176813,7 @@ SQLITE_API int sqlite3_config(int op, ..
   */
   if( sqlite3GlobalConfig.isInit ){
     static const u64 mAnytimeConfigOption = 0
-       | MASKBIT64( SQLITE_CONFIG_LOG )
+/*       | MASKBIT64( SQLITE_CONFIG_LOG )*/
        | MASKBIT64( SQLITE_CONFIG_PCACHE_HDRSZ )
     ;
     if( op<0 || op>63 || (MASKBIT64(op) & mAnytimeConfigOption)==0 ){
